src/pytorch_lightning/utilities/apply_func.py

```diff
@@ -76,7 +76,7 @@ def apply_to_collection(
     dtype: Union[type, Any, Tuple[Union[type, Any]]],
     function: Callable,
     *args: Any,
-    wrong_dtype: Optional[Union[type, Tuple[type, ...]]] = None,
+    wrong_dtype: Optional[Union[type, Tuple[type], Tuple]] = None,
```

Comment:
Let's revert this. I don't see any errors locally. This must have been due to a conflict with changes from master.